keps/sig-network/0015-20180614-SCTP-support.md

```diff
@@ -0,0 +1,291 @@
+SCTP is an IETF protocol specified in [RFC4960][], and it is used widely in telecommunications network stacks.
+Once SCTP support is added as a new protocol option for Service and NetworkPolicy those applications that require SCTP as L4 protocol on their interfaces can be deployed on Kubernetes clusters on a more straightforward way. For example they can use the native kube-dns based service discovery, and their communication can be controlled on the native NetworkPolicy way.
+
+<!---
+The `Summary` section is incredibly important for producing high quality user focused documentation such as release notes or a development road map.
+It should be possible to collect this information before implementation begins in order to avoid requiring implementors to split their attention between writing release notes and implementing the feature itself.
+KEP editors, SIG Docs, and SIG PM should help to ensure that the tone and content of the `Summary` section is useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+--->
+
+[Service]: https://kubernetes.io/docs/concepts/services-networking/service/
+[NetworkPolicy]: https://kubernetes.io/docs/concepts/services-networking/network-policies/
+[RFC4960]: https://tools.ietf.org/html/rfc4960
+
+## Motivation
+
+SCTP is a widely used protocol in telecommunications. It would ease the management and execution of telecommunication applications on Kubernetes if SCTP were added as a protocol option to Kubernetes Service and NetworkPolicy. 
+
+<!---
+This section is for explicitly listing the motivation, goals and non-goals of this KEP.
+Describe why the change is important and the benefits to users.
+The motivation section can optionally provide links to [experience reports][] to demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+--->
+### Goals
+
+Add SCTP support to Kubernetes Service and NetworkPolicy, so applications running in pods can use the native kube-dns based service discovery for SCTP based services, and their communication can be controlled via the native NetworkPolicy way.
+
+<!---
+List the specific goals of the KEP.
+How will we know that this has succeeded?
+--->
+
+### Non-Goals
+
+It is not a goal here to add SCTP support to load balancers that are provided by cloud providers. I.e. the Kubernetes user can define Services with type=LoadBalancer and Protocol=SCTP, but if the actual load balancer implementation does not support SCTP then the creation of the Service/load balancer fails.
+
+<!---
+What is out of scope for his KEP?
+Listing non-goals helps to focus discussion and make progress.
+--->
+## Proposal
+
+<!---
+This is where we get down to the nitty gritty of what the proposal actually is.
+--->
+### User Stories [optional]
```

Comment:
I am curious to learn the user stories of having SCTP support?

Reply:
Do you mean, why SCTP support is requested at all?